install/rc.yaml

```diff
@@ -23,7 +23,6 @@ spec:
       containers:
       - name: brokersdk
         image: brokersdk
-        imagePullPolicy: Never
```

Comment:
i'm afraid this appears to be defaulting to a pull policy of "always" now.  sorry.  I could swear k8s only did that if the tag was explicitly ":latest".  Anyway please set it to IfNotPresent